examples/seismic/accuracy.ipynb

```diff
@@ -0,0 +1,433 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "from examples.seismic.acoustic import AcousticWaveSolver\n",
+    "from examples.seismic import Model, RickerSource, Receiver\n",
+    "from devito import clear_cache\n",
+    "from devito.logger import set_log_level\n",
+    "from opescibench import LinePlotter\n",
+    "\n",
+    "import matplotlib.pyplot as plt\n",
+    "from matplotlib import cm"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Model with fixed time step value\n",
+    "class ModelBench(Model):\n",
+    "    \"\"\"\n",
+    "    Physical model used for accuracy benchmarking,\n",
+    "    The critical dt is made small enough to ignore\n",
+    "    time discretization errors\n",
+    "    \"\"\"\n",
+    "\n",
+    "    @property\n",
+    "    def critical_dt(self):\n",
+    "        \"\"\"Critical computational time step value.\"\"\"\n",
+    "        return .1"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Verification\n",
+    "\n",
+    "We compute the error between the numerical and reference solutions for varying spatial discretization order and grid spacing. We also compare the time to solution to the error for these parameters."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Discretization order\n",
+    "orders = (2, 4, 6, 8, 10)\n",
+    "norder = len(orders)\n",
+    "\n",
+    "# Domain sizes and gird spacing\n",
+    "shapes = ((201, 2.0), (161, 2.5), (101, 4.0))\n",
+    "dx = [2.0, 2.5, 4.0]\n",
+    "nshapes = len(shapes)\n",
+    "\n",
+    "# Number of time steps\n",
+    "nt = 1501\n",
+    "\n",
+    "# Time axis\n",
+    "\n",
+    "time = np.linspace(0., 150., nt)\n",
+    "# Source peak frequency\n",
+    "f0 = .070"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Fine grid model\n",
+    "c0 = 1.500\n",
+    "model = ModelBench(vp=c0, origin=(0., 0.), spacing=(1.0, 1.0), shape=(401, 401), nbpml=40, dtype=np.float64)\n"
```

Comment:
is float64 really what you want here?

Reply:
Yes, want to get enough numerical precision for the order of errors below.